backend/coins/btc/handlers/handlers.go

```diff
@@ -288,7 +289,11 @@ func (input *sendTxInput) UnmarshalJSON(jsonBytes []byte) error {
 	if err != nil {
 		return errp.WithMessage(err, "Failed to retrieve fee target code")
 	}
-	input.FeePerKb = btcutil.Amount(jsonBody.FeePerByte * 1000)
+	feePerKb, err := strconv.ParseFloat(jsonBody.FeePerByte, 64)
+	if err != nil {
+		return err
```

Comment:
A better option is probably to set `feePerKb` to 0 and continue. Otherwise, the app's state on the UI becomes inconsistent.